main.tf

```diff
@@ -58,8 +58,9 @@ resource "aws_dynamodb_table" "this" {
     for_each = var.replica_regions
 
     content {
-      region_name = replica.value.region_name
-      kms_key_arn = lookup(replica.value, "kms_key_arn", null)
+      region_name    = replica.value.region_name
+      kms_key_arn    = lookup(replica.value, "kms_key_arn", null)
+      propagate_tags = lookup(replica.value, "propagate_tags", false)
```

Comment:
Usually, setting null as the default is preferred to avoid changes for existing users. Please change it.

Reply:
Please review now.